src/messagesStore/selectors.js

```diff
@@ -0,0 +1,9 @@
+export function getMessages(state) {
+  return state.messages;
+}
+export function getIsLoading(state) {
+  return state.isLoading;
+}
```

Comment:
As we agreed, please only add selectors only as you need them. "The best code is no code at all".